siliconcompiler/schema/schema_cfg.py

```diff
@@ -2127,7 +2134,8 @@ def schema_option(cfg):
                 "api: chip.set('option', 'loglevel', 'INFO')"],
             schelp="""
             Provides explicit control over the level of debug logging printed.
-            Valid entries include INFO, DEBUG, WARNING, ERROR.""")
+            Valid entries include INFO, DEBUG, WARNING, ERROR.""",
+            enum=["INFO", "DEBUG", "WARNING", "ERROR"])
```

Comment:
Was that the last one, I seem to remember there were more special strings?

Reply:
I can double check, I just copied from the line above